app/models/form.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Form, associated with an event
+class Form < ApplicationRecord
+  alias_attribute :id, :form_id
+
+  belongs_to :event, primary_key: :form_master_num,
+                     foreign_key: :form_master_num, inverse_of: :forms
+
+  default_scope { active.within_range }
+  scope :active, -> { where(status_id: 1) }
+
+  scope :within_range, lambda {
+    where('? between effective_start and effective_end', Date.today.to_s)
```

Comment:
Since these are `date` columns in the db, will this query work without casting `Date.today` to a string? Basically, can `Date.today.to_s` be `Date.today`?

Reply:
Yes, you are right about that.

Reply:
yes, you are right about that.